test/MUXControls.Test/MUXControls.Test.Shared.targets

```diff
@@ -34,4 +33,5 @@
   <Import Project="..\..\dev\Interactions\ButtonInteraction\InteractionTests\ButtonInteraction_InteractionTests.projitems" Label="Shared" Condition="$(FeatureInteractionsEnabled) == 'true'" />
   <Import Project="..\..\dev\Interactions\SliderInteraction\InteractionTests\SliderInteraction_InteractionTests.projitems" Label="Shared" Condition="$(FeatureInteractionsEnabled) == 'true'" />
   <Import Project="..\..\dev\TabView\InteractionTests\TabView_InteractionTests.projitems" Label="Shared" Condition="$(FeatureTabViewEnabled) == 'true'" />
+  <Import Project="$(MSBuildThisFileDirectory)\..\..\dev\ProgressBar\InteractionTests\ProgressBar_InteractionTests.projitems" Label="Shared" Condition="$(FeatureProgressBarEnabled) == 'true'" />
```

Comment:
>MSBuildThisFileDirectory [](start = 21, length = 24)

I'm not sure what the MSBuildThisFileDirectory is, or where it came from but I suspect it should be deleted

Reply:
Sorry, I think you need this line, but not the MSBuildThisFileDirectory tag at the start. so just 
`<Import Project="..\..\dev\ProgressBar\InteractionTests\ProgressBar_InteractionTests.projitems" Label="Shared" Condition="$(FeatureProgressBarEnabled) == 'true'" />

---
In reply to: [335752143](https://github.com/microsoft/microsoft-ui-xaml/pull/1453#discussion_r335752143) [](ancestors = 335752143)